src/utils/date-validation.ts

```diff
@@ -0,0 +1,9 @@
+export const validateDate = (date: string) => {
+        const dateList = date.split('/');
+        date = dateList[2]+'-'+dateList[1]+'-'+dateList[0]
+        const pickedDate = new Date(Date.parse(date));
```

Comment:
aqui é realmente necessário usar o Date.parse? n daria pra fazer direto?
```suggestion
        const pickedDate = new Date(date);
```

Reply:
de fato. feito